### ISSUE TITLE
menu + old RLV plugin

### ISSUE THREAD

**Issue by LeonaMorro** [HowardBaxton/nPose#34]
while digging through the code I noticed the following in DoMenu_AccessCtrl. 

```
    else if((Permissions == "GROUP" && llSameGroup(toucher))
     | (Permissions == "PUBLIC") 
     | (!~llListFindList(victims, [(string)toucher]))) {
        authorized = TRUE;
    }
```

I guess this is wrong. I expect:
- the menu is shown to "victims" (if menu permission is set to GOUP or PUBLIC)
- menu permission GROUP does not work
- menu Permission OWNER does not work

I didn't test it till now.

**Comment by LeonaMorro** [comment]
I saw you are already working on this (even if the mentioned branch currently isn't in this repo) so I assign this to you.

**Comment by HowardBaxton** [comment]
Your branch also addresses this issue and some more.  I have been waiting for your pull request.

**Comment by LeonaMorro** [comment]
oh .. yes thats right. But I wasn't sure if I should made a pull request, because there are "complex" changes in my branch and I wasn't sure if we should/are willing to implement them currenly. Perhaps we should disuss that first.

**Comment by HowardBaxton** [comment]
I think we should make the commit to remove the DoMenu_AccessCtrl and put it into Dialog function within the menu, as well the forwarding of link messages to DoMenu.  

The other things in your branch are what you're wanting to discuss if I understand correctly.

**Comment by LeonaMorro** [comment]
Yes, that are huge changes (also removing the DoMenu_AccessCtrl maybe a huge change). So we have to be sure if we 1) want this changes and 2) want the changes currently

**Comment by LeonaMorro** [comment]
fixed with f6f6aa60d962d6f592423753d4595ca141ce355b
